crates/core/src/host/wasmtime/mod.rs

```diff
@@ -91,34 +91,24 @@ struct WasmtimeFuel(u64);
 
 impl WasmtimeFuel {
     /// 1000 energy quanta == 1 wasmtime fuel unit
-    const QUANTA_MULTIPLIER: i128 = 1_000;
-
-    /// Convert from EnergyQuanta to wasmtime fuel. The second element of the tuple is the remainder
-    /// of quanta in the case that `eq` isn't in u64 range. It can be safely ignored if the supplied
-    /// budget isn't being bookkept.
-    fn from_energy_quanta(eq: EnergyQuanta) -> (Self, EnergyQuanta) {
```

Comment:
It doesn't look like the below logic has been moved anywhere else. Why?

Reply:
with ReducerBudget, this conversion logic is unnecessary